include/pybind11/chrono.h

```diff
@@ -99,15 +99,15 @@ class duration_caster {
     PYBIND11_TYPE_CASTER(type, const_name("datetime.timedelta"));
 };
 
-inline std::tm *localtime_thread_safe(const std::time_t *time, std::tm *buf) {
+inline std::tm *gmtime_thread_safe(const std::time_t *time, std::tm *buf) {
 #if (defined(__STDC_LIB_EXT1__) && defined(__STDC_WANT_LIB_EXT1__)) || defined(_MSC_VER)
     if (localtime_s(buf, time))
```

Comment:
All the Windows tests are failing. Do you already have a plan for that? Could we simply remove the `#if` and use the same code, with `mutex`, everywhere?